omod/src/main/java/org/openmrs/module/registrationapp/RegistrationAppUiUtils.java

```diff
@@ -61,7 +61,22 @@ public String getAttribute(Person person, String attributeTypeUuid) {
 		
 		return null;
 	}
-	
+	/**
+	 * Gets the concept name  for the specified coded concept person attribute on registration
```

Comment:
It would also be great to add a since annotation to this method. https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NewPublicMethodsandClasses

Reply:
Basing on the method description, does this mean that the method cannot be used on anything that is not registration?

Reply:
> 
> 
> Basing on the method description, does this mean that the method cannot be used on anything that is not registration?

The method can work in any other place, however, we realized its urgent need in regards to registration, but it can be used anywhere whenever need arises

Reply:
> 
> 
> Is this method dealing with only concept attributes? Or any person attribute even if it is not concept?

The method works specifically for concept attributes. This is because all other person attributes could display as needed except the concept attributes

Reply:
> 
> 
> It would also be great to add a since annotation to this method. https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NewPublicMethodsandClasses

The annotation has been added

Reply:
So what happens when the attribute type uuid is pointing to something else like location, which is not a concept?

Reply:
If it can be used anywhere besides registration, do we still need to bind the method description to registration?

Reply:
> 
> 
> So what happens when the attribute type uuid is pointing to something else like location, which is not a concept?
The getAttribute Method which is already defined  will handle that particular field

Reply:
> 
> 
> If it can be used anywhere besides registration, do we still need to bind the method description to registration?

I have changed this description

Reply:
> 
> 
> So what happens when the attribute type uuid is pointing to something else like location, which is not a concept?

@dkayiwa i have tested it with other attribute type uuids, it will still display their values as required. as long the field is a person attribute, its just that my foccus was on concept values

Reply:
This therefore means that the description should be generic enough not to be restricted to concepts.